zds/member/views/reports.py

```diff
@@ -49,3 +52,26 @@ def post(self, request, *args, **kwargs):
             alert.solve(request.user)
         messages.success(request, _("Merci, l'alerte a bien été résolue."))
         return redirect(alert.profile.get_absolute_url())
+
+
+class PotentialObsoleteListView(LoginRequiredMixin, PermissionRequiredMixin, ZdSPagingListView):
+    """
+    Displays a paginated list of potential obsolete reports.
+    Only accessible to logged-in users.
+    """
+
+    permission_required = "tutorialv2.change_publishablecontent"
+    model = PotentialObsolete
+    template_name = "member/potentialobsolete_list.html"  # Ensure this template exists
+    context_object_name = "reports"
+    ordering = "-report_date"
+    paginate_by = 10  # Enable pagination
+
+    def get_queryset(self):
+        """Customize the queryset to order by report_date (most recent first)."""
+        return PotentialObsolete.objects.all().order_by(self.ordering)
```

Comment:
```suggestion
        return PotentialObsolete.objects.all().select_related("published_content").order_by(self.ordering)
```
(pour faire une jointure avec les published_content et éviter N+1 requêtes)